fiddy/derivative_check.py

```diff
@@ -161,3 +162,109 @@ def method(self, *args, **kwargs):
             success=success,
         )
         return derivative_check_result
+
+
+class HybridDerivativeCheck(DerivativeCheck):
+    method_id = "hybrid"
```

Comment:
Added. LMK if more is needed.

Reply:
Ah, I meant the actual formula, preferably with latex. There's an example of latex in RTD for `amici.import_utils.noise_distribution_to_cost_function` here [1], with the docstring here [2].

But this can also wait until you publish the formula. If you decide to wait, could you add a TODO to do this later? Would be great to add a reference to your paper when it's published, too.
 
 [1] https://amici.readthedocs.io/en/latest/generated/amici.import_utils.html#amici.import_utils.noise_distribution_to_cost_function
[2] https://github.com/AMICI-dev/AMICI/blob/3c5e997df3655c26dde35705ef25b2a0f419fe8b/python/sdist/amici/import_utils.py#L105-L107